app/src/main/java/org/covidwatch/android/ui/ItemBinding.kt

```diff
@@ -13,4 +14,11 @@ object ItemBindings {
             BR.item,
             R.layout.item_exposure_info
         )
+
+    @JvmStatic
+    fun bind(viewModel: ExposuresViewModel): ItemBinding<CovidExposureInformation> =
```

Comment:
What is this for?

Reply:
This is for this library https://github.com/evant/binding-collection-adapter

Again, I introduced this in the `ag-minimal` app as it allows to write list-type screen with minimum of code using data binding.